demisto_sdk/commands/openapi_codegen/openapi_codegen.py

```diff
@@ -0,0 +1,837 @@
+                                        new_ref_arg['properties'][ck] = {}
+                                        new_ref_arg['properties'][ck]['type'] = cv.get('type', 'string')
+                                        new_ref_arg['properties'][ck]['description'] = cv.get('description', '')
+                                        new_ref_arg['properties'][ck]['required'] = True \
+                                            if ck in self.reference.get(c_ref, {}).get('required', []) else False
+                                        new_ref_arg['properties'][ck]['options'] = [str(x) for x in cv.get('enum')] \
+                                            if arg.get('enum', None) else None
+                            else:
+                                new_ref_arg.update(v)
+                            new_ref_arg['ref'] = ref
+                            new_function['arguments'].append(self.init_arg(new_ref_arg))
+            if not refs:
+                if arg.get('schema', {}).get('type'):
+                    arg['type'] = arg['schema']['type']
+                new_function['arguments'].append(self.init_arg(arg))
+
+    def add_function(self, path, method, data, params):
+        new_function = {}
+        new_function['path'] = '/'.join(x.split(' ')[0] for x in path.split('/')).strip('/')
+        new_function['method'] = method
+        name = data.get('operationId', None)
+        if not name:
+            try:
+                name = data.get('summary', None)
+                name = name.replace(" ", "-")
+            except Exception:
+                name = None
+        if not name:
+            name = '_'.join([re.sub(r'\{[^)]*\}', '', x) for x in path.split('/')])
+        name = self.clean_function_name(name)
+        new_function['name'] = name
+        func_desc = data.get('summary', None)
+        if not func_desc:
+            func_desc = data.get('description', '')
+        func_desc = self.clean_description(func_desc)
+        new_function['description'] = func_desc
+        new_function['execution'] = False
+        new_function['arguments'] = []
+        new_function['parameters'] = data.get('parameters', None)
+        new_function['consumes'] = data.get('consumes', [])
+        new_function['produces'] = data.get('produces', [])
+        new_function['outputs'] = []
+        new_function['responses'] = []
+        if not new_function['parameters']:
+            new_function['parameters'] = params
+            iter_item = params
+        else:
+            iter_item = data.get('parameters', [])
+        self.extract_args(iter_item, new_function)
+        self.extract_outputs(data, new_function)
+
+        self.functions.append(new_function)
+
+    def load_file(self, file_path):
+        error = None
+        try:
+            self.json = json.load(open(file_path, 'rb'))
+            self.file_load = True
+        except Exception as err:
+            error = err

```

Comment:
isnt this always overridden by line 273 ?

Reply:
nope, self.file_load will be `False`